src/library_glemu.js

```diff
@@ -1499,58 +1499,29 @@ var LibraryGLEmulation = {
         var lines = null;
         switch (combiner) {
           case GL_REPLACE: {
-            var line = [
-              outputType + " " + outputVar,
-              " = ",
-                src0Expr,
-              ";",
-            ];
-            lines = [line.join("")];
+            lines = [`${outputType} ${outputVar} = ${src0Expr};`]
             break;
           }
           case GL_MODULATE: {
-            var line = [
-              outputType + " " + outputVar + " = ",
-                src0Expr + " * " + src1Expr,
-              ";",
-            ];
-            lines = [line.join("")];
+            lines = [`${outputType} ${outputVar} = ${src0Expr} * ${src1Expr};`];
             break;
           }
           case GL_ADD: {
-            var line = [
-              outputType + " " + outputVar + " = ",
-                src0Expr + " + " + src1Expr,
-              ";",
-            ];
-            lines = [line.join("")];
+            lines = [`${outputType} ${outputVar} = ${src0Expr} + ${src1Expr};`]
             break;
           }
           case GL_SUBTRACT: {
-            var line = [
-              outputType + " " + outputVar + " = ",
-                src0Expr + " - " + src1Expr,
-              ";",
-            ];
-            lines = [line.join("")];
+            lines = [`${outputType} ${outputVar} = ${src0Expr} - ${src1Expr};`]
             break;
           }
           case GL_INTERPOLATE: {
-            var prefix = TEXENVJIT_NAMESPACE_PREFIX + 'env' + texUnitID + "_";
+            var prefix = `${TEXENVJIT_NAMESPACE_PREFIX}env${texUnitID}_`;
             var arg2Var = prefix + "colorSrc2";
```

Comment:
```suggestion
            var arg2Var = `${prefix} colorSrc2`;
```

Reply:
There is no space so I figured this was simple concatenation, but update for consistency with surrounding code.